ci/azure-pipelines-nightly.yml

```diff
@@ -6,8 +6,8 @@
 # Documentation at https://aka.ms/yaml
 
 schedules:
-# For testing purposes, run this pipeline every day at 07:21 UTC
-# Use a random minute value to avoid congestion on the hour
+# Run this pipeline every day at 07:21 UTC.
+# Use an arbitrary minute value to avoid congestion on the hour.
```

Comment:
Regarding https://github.com/lowRISC/opentitan/pull/16813#discussion_r1050053510, Azure's [cron-style specification](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml#cron-syntax) doesn't seem to allow dynamically choosing a random value. In any case, this is not critical for this purpose, so I've updated the comment to say an *arbitrary* rather than *random* minute.